welcome/learn_more_about_openshift.adoc

```diff
@@ -134,34 +145,34 @@ a|* xref:../machine_management/index.adoc#machine-api-overview_overview-of-machi
 | Manage xref:../machine_management/index.adoc#machine-mgmt-intro-managing-compute_overview-of-machine-management[compute] and xref:../machine_management/index.adoc#machine-mgmt-intro-managing-control-plane_overview-of-machine-management[control plane] machines with machine sets.
 |
 
-|xref:../machine_management/deploying-machine-health-checks.adoc#deploying-machine-health-checks[Deploying machine health checks]
-|
+| xref:../machine_management/deploying-machine-health-checks.adoc#deploying-machine-health-checks[Deploying machine health checks]
+| xref:../machine_management/control_plane_machine_management/cpmso-about.adoc#cpmso-about[About control plane machine sets]
 
-|xref:../machine_management/applying-autoscaling.adoc#applying-autoscaling[Applying autoscaling to an {product-title} cluster]
-|
+| xref:../machine_management/applying-autoscaling.adoc#applying-autoscaling[Applying autoscaling to an {product-title} cluster]
+| xref:../nodes/pods/nodes-pods-priority.adoc#nodes-pods-priority[Including pod priority in pod scheduling decisions]
 
 | xref:../registry/index.adoc#registry-overview[Manage container registries]
 | link:https://access.redhat.com/documentation/en-us/red_hat_quay/[Red Hat Quay]
 
 | xref:../authentication/understanding-authentication.adoc#understanding-authentication[Manage users and groups]
-|
+| xref:../authentication/impersonating-system-admin.adoc#impersonating-system-admin[Impersonating the system:admin user]
 
 | xref:../authentication/understanding-authentication.adoc#understanding-authentication[Manage authentication]
 | xref:../authentication/understanding-identity-provider.adoc#supported-identity-providers[multiple identity providers]
 
 | Manage xref:../security/certificates/replacing-default-ingress-certificate.adoc#replacing-default-ingress[ingress], xref:../security/certificates/api-server.adoc#api-server-certificates[API server], and xref:../security/certificates/service-serving-certificate.adoc#add-service-serving[service] certificates
-|
+| xref:../networking/network_security/network-policy-apis.adoc#network-policy-apis[network security]
```

Comment:
```suggestion
| xref:../networking/network_security/network-policy-apis.adoc#network-policy-apis[Network security]
```

I see other entries that start with a lowercase character. Please update these link headings as well to start with an uppercase character.